docs/architecture/modernize-desktop/deploy-modern-applications.md

```diff
@@ -0,0 +1,220 @@
+
+#### OS managed
+
+The OS handles all the processes for installing, updating, and removing an application. Applications are installed per user but downloaded only once, minimizing the disk footprint. Microsoft is working on providing the MSIX experience also on Windows 7.
+
+#### Windows provides integrity for the app
+
+With the use of digital signatures, you can guarantee that you don't install an application from untrusted sources. MSIX also prevents tampering because:
+
+- It keeps a record of file hashes.
+- It detects if a file has been modified after installation.
+
+#### Works for the entire App Catalog
+
+One of the coolest things about MSIX is that it works for the entire application catalog, Windows Forms, WPF, MFC/ATL, Delphi, even if you want to do xCopy deployment, ClickOnce, or going to the Store, you can use the same MSIX package.
+
+### Tools
+
+#### Windows Application Packaging Project
+
+You can use the **Windows Application Packaging Project** project in Visual Studio to generate a package for your desktop app. Then, you can publish that package to the Microsoft Store or sideload it onto one or more PCs.
+
+#### MSIX Packaging Tool
+
+The MSIX Packaging Tool enables you to repackage your existing Win32 applications to the MSIX format. It offers both an interactive UI and a command line for conversions and gives you the ability to convert an application without having the source code.
+
+![MSIX Packaging Tool](media\deploying-modern-desktop-applications\msix-packaging-tool.png)
+
+#### Package Support Framework
+
+The Package Support Framework is an open-source kit that helps you apply fixes to your existing Win32 application when you don't have access to the source code, so that it can run in an MSIX container. The Package Support Framework helps your application follow the best practices of the modern runtime environment.
+
+#### App Installer
+
+App Installer allows Windows 10 apps to be installed by double-clicking the app package. This means that users don't need to use PowerShell or other developer tools to deploy Windows 10 apps. The App Installer can also install an app from the web, optional packages, and related sets.
+
+## How to create an MSIX package from an existing Win32 desktop application
+
+Let's go through the process to create an MSIX package from an existing Win32 application. In this example, we'll use a Windows Forms app.
+
+To start, add a new project to your solution, select the Windows Application Packaging Project and give it a name.
+
+![Adding new Windows Application Packaging Project](media\deploying-modern-desktop-applications\adding-packaging-project.png)
+
+You'll see the structure of the packaging project and note a special folder called Applications. Inside this folder you can specify which applications you want to include in the package, it can be more than one.
```

Comment:
```suggestion
You'll see the structure of the packaging project and note a special folder called *Applications*. Inside this folder, you can specify which applications you want to include in the package. It can be more than one.
```